ConcernsCaseWork/ConcernsCaseWork/Program.cs

```diff
@@ -10,12 +10,17 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            Log.Logger = new LoggerConfiguration()
+            var logConfig = new LoggerConfiguration()
                 .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
                 .Enrich.FromLogContext()
                 .WriteTo.Console(new RenderedCompactJsonFormatter())
-				.WriteTo.Sentry()
-                .CreateLogger();
+				.WriteTo.Sentry();
+                
+	        #if DEBUG
```

Comment:
QUERY are these debug statements meant to be here?

Reply:
can't remember why so removed it